docs/getting_started.rst

```diff
@@ -22,12 +22,18 @@ Simple JWT can be installed with pip::
   pip install djangorestframework-simplejwt
 
 Then, your django project must be configured to use the library.  In
-``settings.py``, add
+``settings.py``, add ``rest_framework_simplejwt`` to the list of apps and
 ``rest_framework_simplejwt.authentication.JWTAuthentication`` to the list of
 authentication classes:
 
 .. code-block:: python
 
+  INSTALLED_APPS = [
+      ...
+      'rest_framework_simplejwt',
+      ...
+  ]
+
```

Comment:
This really shouldn't be here. Many people don't enable localization. Where you see "You can also include a route for Simple JWT’s TokenVerifyView", write some docs below that section dedicated for "adding localizations/translations" (add both words for searching sake in RTD).